packages/neuron-wallet/package.json

```diff
@@ -36,8 +36,9 @@
     ]
   },
   "dependencies": {
-    "@ckb-lumos/base": "0.18.0-rc2",
-    "@ckb-lumos/rpc": "0.18.0-rc2",
+    "@ckb-lumos/base": "0.20.0-alpha.3",
+    "@ckb-lumos/rpc": "0.20.0-alpha.3",
+    "@ckb-lumos/ckb-indexer": "0.20.0-alpha.3",
```

Comment:
It appears to be added manually because the order is not alphanumeric

Reply:
refactored